docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -388,54 +439,20 @@ _invalid_ quantities are `0.5` and `1500m`.
 Users can use any domain name prefix other than "`kubernetes.io`" which is reserved.
 {: .note}
 
-There are two steps required to use Extended Resources. First, the
-cluster operator must advertise a per-node Extended Resource on one or more
-nodes. Second, users must request the Extended Resource in Pods.
-
-To advertise a new Extended Resource, the cluster operator should
-submit a `PATCH` HTTP request to the API server to specify the available
-quantity in the `status.capacity` for a node in the cluster. After this
-operation, the node's `status.capacity` will include a new resource. The
-`status.allocatable` field is updated automatically with the new resource
-asynchronously by the kubelet. Note that because the scheduler uses the
-node `status.allocatable` value when evaluating Pod fitness, there may
-be a short delay between patching the node capacity with a new resource and the
-first pod that requests the resource to be scheduled on that node.
-
-**Example:**
-
-Here is an example showing how to use `curl` to form an HTTP request that
-advertises five "example.com/foo" resources on node `k8s-node-1` whose master
-is `k8s-master`.
-
-```shell
-curl --header "Content-Type: application/json-patch+json" \
---request PATCH \
---data '[{"op": "add", "path": "/status/capacity/example.com~1foo", "value": "5"}]' \
-http://k8s-master:8080/api/v1/nodes/k8s-node-1/status
-```
-
-**Note**: In the preceding request, `~1` is the encoding for the character `/`
-in the patch path. The operation path value in JSON-Patch is interpreted as a
-JSON-Pointer. For more details, see
-[IETF RFC 6901, section 3](https://tools.ietf.org/html/rfc6901#section-3).
-{: .note}
-
```

Comment:
Line 391-423 above is about advertising an extended resource given that these resources are not discoverable by Kubernetes. Why are we deleting this content?

Reply:
I think it should be the Device Plugin that advertises the node-level resources. If there's a legit use case of directly patching node objects with HTTP requests, let me know. I will add this back.

Reply:
In the previous paragraphs, it is mentioned that software licenses can be managed as extended resources, no?
I am not aware of any efforts to support that using device plugins.